src/main/resources/db/migration/V20211018_12841__RDM-12841_add_case_link_table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE public.case_link (
+    case_id bigint NOT NULL,
+    linked_case_id bigint NOT NULL,
```

Comment:
The case data is store in a table called `case_data`.  All other tables that link to this property use `case_data_id` so should this.

Reply:
This naming reflects HLD and is already referenced in ccd-case-disposer so ignore comment.